File: 4st/Seohyun/programmer/머쓱이보다 키 큰 사람.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// array_len은 배열 array의 길이입니다.
int solution(int array[], size_t array_len, int height) {
    int answer=0;
    for(int i=0; i<array_len; i++) {
        if(array[i]>height) answer++;
    }
    return answer;
}

File: 4st/Eunseong/머쓱이보다 키 큰 사람.java
class Solution {
    public int solution(int[] array, int height) {
        int answer = 0;
        
        for (int i = 0; i <= array.length - 1; i++) {
            if (array[i] > height) {
                answer++;
            }
        }
        
        return answer;
    }
}